src/app/controllers/censusdata/create.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
-flnd.birdSpeciesList = {
-    retrieveRecords: function(config, $scope, messenger, authService) {
+flnd.censusDataCreate = {
+    retrieveBirds: function(config, $scope, messenger, authService) {
```

Comment:
@kpunjani this was causing a problem for the Bird Species management page, where the controller already had a function in the `flnd.birdSpeciesList` namespace. That function was being replaced by this on from the census data. To fix this, I merged the two namespaces here in `create.js`, and renamed `retrieveRecords` to the more specific name `retrieveBirds`.